<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactDetails</class>
 <widget class="QMainWindow" name="ContactDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/assets/assets/favicon.png</normaloff>:/assets/assets/favicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">backgorund: #FFF;

padding: 0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="ContainerImage">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>52</y>
      <width>491</width>
      <height>279</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFF;
border: none;
outline: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="pixmap">
        <pixmap resource="assets.qrc">:/assets/assets/person-circle.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="NavigatorBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background: #241F31; */
background-color: #E0E0E0;
border: none;
outline: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="backPutton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 10px;
background: none;
border: none;
outline: none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets.qrc">
         <normaloff>:/assets/assets/arrow-left-small_b.svg</normaloff>:/assets/assets/arrow-left-small_b.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleSection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: none;
border: none;
outline: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>324</y>
      <width>491</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFF;
border: none;

outline: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="ButtonSave">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #241F31;
color: #FFF;
height: 50px;

font-size: 20px;
font-weight: 700;</string>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDay">
       <property name="styleSheet">
        <string notr="true">color: #6A6A6A;
background: none;
font-weight: 400;
font-size: 14px;

padding: 0;
</string>
       </property>
       <property name="text">
        <string>Dia Aniversário</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSpinBox" name="spinBoxDay">
       <property name="styleSheet">
        <string notr="true">background-color: #E0E0E0;
height: 35px;
padding: 15px;

font-size: 22px;
font-weight: 700;</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>31</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEditName">
       <property name="styleSheet">
        <string notr="true">background-color: #E0E0E0;
height: 35px;
padding: 15px;

font-size: 17px;
font-weight: 700;

margin-bottom: 20px;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>133</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0" colspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>470</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelMonth">
       <property name="styleSheet">
        <string notr="true">color: #6A6A6A;
background: none;
font-weight: 400;
font-size: 14px;

padding: 0;</string>
       </property>
       <property name="text">
        <string>Mês Aniversário</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelName">
       <property name="styleSheet">
        <string notr="true">color: #6A6A6A;
background: none;
font-weight: 400;
font-size: 14px;

padding: 0;</string>
       </property>
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBoxMonth">
       <property name="styleSheet">
        <string notr="true">background-color: #E0E0E0;
height: 35px;
padding: 15px;

font-size: 22px;
font-weight: 700;</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>backPutton</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>spinBoxDay</tabstop>
  <tabstop>spinBoxMonth</tabstop>
  <tabstop>ButtonSave</tabstop>
 </tabstops>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
